### ISSUE TITLE
REPL dependency check for Readline

### ISSUE THREAD

**Issue by ToddAndMargo** [nxadm/rakudo-pkg#60]
Dear nxadm,

Fedora 31
https://github.com/nxadm/rakudo-pkg/releases

$ rpm -qa *rakudo*
rakudo-pkg-2019.07.1-03.x86_64

Would you please consider the following Request for Enhancement?

When running REPL and Readline has not been installed, when you use your arrow keys, which I use a lot, you get a lot of

my $history^[[A^[[A
a todd special booboo^[[D^[[D^[[D^[[D^[[D

Would you please consider informing the user that Readline is missing and give zef install Readline as an example of how to install it.

Maybe even consider adding Readline to the package!

Thank you for your consideration,
-T

**Comment by nxadm** [comment]
Thanks, @ToddAndMargo, I'll leave the issue open while I think about it. Your input is of course welcome.

The only problem I have is that both readline and linenoise integration have been rather spotty with their binary dependencies, so If we do it, we have to make sure we do it on a maintainable way.

**Comment by ToddAndMargo** [comment]
On 2020-01-26 03:48, nxadm wrote:
> Thanks, @ToddAndMargo <https://github.com/ToddAndMargo>, I'll leave the 
> issue open while I think about it. Your input is of course welcome.
> 
> The only problem I have is that both readline and linenoise integration 
> have been rather spotty with their binary dependencies, so If we do it, 
> we have to make sure we do it on a maintainable way.

That is the best that can be expected.  Thank you!

**Comment by tbrowder** [comment]
I just installed the package (on Debian) and when I went into the REPL the first time I got a message about installing either Readline or Linenoise with zef.

At first I blindly tried that inside the REPL.  I should have known better, but a person new to Raku may not know that. After I left the REPL, installed Readline, and re-entered the REPL, there was no more message.

Great packaging, @nxadm! Thanks so much. I now have my four active servers set up so keeping up with future releases will be a breeze thanks to your efforts. Previously I was installing from source, a bit of a pain for my old, home-grown solution, so I hadn't upgraded since early last year!
